src/main/webapp/components/search-forms/container.tsx

```diff
@@ -153,7 +162,11 @@ export default () => {
   }
 
   const forms = getIn(data, ['metacardsByTag', 'attributes'], [])
-
+  const userDefaultForm = getIn(
+    data,
+    ['user', 'preferences', 'querySettings', 'template', 'id'],
+    null
```

Comment:
I think the value of this field will always be null or initialized https://github.com/graphql/graphql-js/issues/1298#issuecomment-377365916